meteor/client/ui/Settings/ShowStyle/VariantSettings.tsx

```diff
@@ -216,7 +216,7 @@ export const ShowStyleVariantsSettings = withTranslation()(
 									<div>
 										<div className="mod mvs mhs">
 											<label className="field">
-												{t('Variant Name')}
+												{t('Name')}
```

Comment:
FYI, this will break the translations for NRK since they don't have a key for `Name`, but they do for `Variant Name`:
```
msgid "Variant Name"
msgstr "Variantnavn"
```
_Above can be found in Core under "meteor/i18n/..."_

Not entirely sure what our best approach here would be besides creating our own translation file. I've written it on the agenda for our joint developer meeting with NRK.

Same comment goes for the other translation.